### PR TITLE
0.21.0

### LINKED ISSUE #2881: Release 0.21
I went through issues and tagged things for the 0.21 release https://github.com/pyodide/pyodide/milestone/11 

Mostly naming related subjects need to be discussed/addressed before the release IMO, since once we release they will be harder to change.

### PR DESCRIPTION
**Pull request by ryanking13**
Close #2881

### CODE REVIEW

File: docs/project/changelog.md [hunk 1]
```diff
@@ -12,7 +12,11 @@ substitutions:
 
 # Change Log
 
-## Unreleased
+## Version 0.21.0
+
+_August XX, 2022_
+
+[See the release notes for a summary.](TODO)
```
Comment (ryanking13) [inline docs/project/changelog.md:19]: This need to be updated

File: docs/project/changelog.md [hunk 2]
```diff
@@ -12,7 +12,11 @@ substitutions:
 
 # Change Log
 
-## Unreleased
+## Version 0.21.0
+
+_August XX, 2022_
```
Comment (ryanking13) [inline docs/project/changelog.md:17]: This need to be updated

File: docs/project/changelog.md (hunk 1 above)
Comment (hoodmane) [reply docs/project/changelog.md:19]: I can try to write some release notes.

Comment (ryanking13) [reply docs/project/changelog.md:19]: Thanks a lot!

**Comment by rth** [comment]
LGTM, thanks! Maybe it would be good to merge https://github.com/pyodide/pyodide/pull/2953 before, but it's not critical.

Any other things left before we can release? cc @hoodmane

**Pull request merged**